config/jobs/kubernetes/test-infra/test-infra-canaries.yaml

```diff
@@ -19,9 +19,9 @@ presubmits:
         - --nobuild_tests_only
         - //...
     annotations:
-      testgrid-create-test-group: 'true'
-      testgrid-dashboards: sig-testing-canaries
-      testgrid-tab-name: test-infra-bazel
+      testgrid-dashboards: sig-testing-canaries, presubmits-test-infra
```

Comment:
comma separated seems kind of lame. Should we just add a `testgrid` field rather than relying on annotations which aren't strictly relevant?